efsity/src/main/java/org/smartregister/command/TranslateCommand.java

```diff
@@ -379,7 +379,12 @@ private void extractContent(
       // Handle the case where inputFilePath is a directory (folders may contain multiple JSON
       // files)
 
-      Files.walk(tempsConfig)
+      Path inputDir;
+      if (extractionType.equals("configs")) {
```

Comment:
Start with the string for the condition here. Usually preferred incase the object is null. i.e. 
```java
"configs".equals(extractionType)
```